src/components/Sidebar.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { ReactChild } from 'react'
+
+import DevicesIcon from 'cozy-ui/transpiled/react/Icons/Devices'
+import GlobeIcon from 'cozy-ui/transpiled/react/Icons/Globe'
+import GraphCircle from 'cozy-ui/transpiled/react/Icons/GraphCircle'
+import HandIcon from 'cozy-ui/transpiled/react/Icons/Hand'
+import List from 'cozy-ui/transpiled/react/MuiCozyTheme/List'
+import Logout from 'cozy-ui/transpiled/react/Icons/Logout'
+import PeopleIcon from 'cozy-ui/transpiled/react/Icons/People'
+import UnknowIcon from 'cozy-ui/transpiled/react/Icons/Unknow'
+import flag from 'cozy-flags'
+import { useI18n } from 'cozy-ui/transpiled/react/I18n'
+
+import { MenuItemAnchor } from 'components/menu/MenuItemAnchor'
+import { MenuItemButton } from 'components/menu/MenuItemButton'
+import { MenuItemNavLink } from 'components/menu/MenuItemNavLink'
+import { MenuList } from 'components/menu/MenuList'
+import { routes } from 'constants/routes'
+import { useDiskPercentage } from 'hooks/useDiskPercentage'
+import { useLogout } from 'hooks/useLogout'
+
+export const Sidebar = (): JSX.Element => {
+  const { t } = useI18n()
+  const percent = useDiskPercentage()
+  const logout = useLogout()
+
+  return (
+    <List>
+      <MenuList title={t('Nav.header_general')}>
+        <MenuItemNavLink
+          to={routes.profile}
+          primary={t('Nav.profile')}
+          icon={PeopleIcon}
+        />
+
+        <MenuItemNavLink
+          to={routes.connectedDevices}
+          primary={t('Nav.connected_devices')}
+          icon={DevicesIcon}
+        />
+
+        <MenuItemNavLink
+          to={routes.sessions}
+          primary={t('Nav.sessions')}
+          icon={GlobeIcon}
+        />
+
+        <MenuItemNavLink
+          to={routes.storage}
+          primary={t('Nav.storage')}
+          secondary={percent && `${percent}% ${t('Nav.secondary_used')}`}
+          icon={GraphCircle}
+        />
+
+        {(flag('settings.permissions-dashboard') as unknown as ReactChild) && (
+          <MenuItemNavLink
+            to={routes.appList}
+            primary={t('Nav.permissions')}
+            icon={HandIcon}
+          />
+        )}
+      </MenuList>
+
+      <MenuList title={t('Nav.header_other')}>
+        <MenuItemAnchor
+          primary={t('Nav.primary_faq')}
+          href={routes.external_faq}
+          target="_blank"
+          icon={UnknowIcon}
+        />
+
+        <MenuItemButton
+          primary={t('Nav.primary_logout')}
+          icon={Logout}
+          onClick={(): void => void logout()}
+        />
+      </MenuList>
+    </List>
+  )
+}
+
+export default Sidebar
```

Comment:
je ne sais pas si ma remarque va être pertinente, mais si on chercher ici à gérer une Sidebar, on utilise plutôt Nav, voir un exemple ici https://github.com/cozy/coachCO2/blob/master/src/components/Sidebar.jsx attention par contre c'est avec la version 5 du routeur, je suis justement en train de corriger cozy-ui pour que ça marche avec le routeur v6

Reply:
Je referai un refactor avec Nav avec la compat router 6. Je suis d'accord sur la sémantique un nav element serait plus adéquat.

Reply:
https://github.com/cozy/cozy-ui/pull/2223#event-7267856942

Here you go ;)

Reply:
J'ai regardé, désolé je ne pourrais pas implémenter ces composants sur cette PR, il y a trop de refactos. Nous n'avions pas parlé de ces éléments quand on a fait le point UI, du coup j'ai tout qui est fait avec les api List, ListItem etc.

J'ai cependant amélioré la sémantique HTML car il y avait en effet des défauts. On a bien maintenant une `nav>ul>li*n`

Pour le reste, j'essaierais de mettre à jour ListItem vers Nav & cie sur les features suivantes, vu qu'elles ajoutent de toute façon des entrées dans le menu.

cc @Crash-- @JF-Cozy

Reply:
> J'ai regardé, désolé je ne pourrais pas implémenter ces composants sur cette PR, il y a trop de refactos. Nous n'avions pas parlé de ces éléments quand on a fait le point UI, du coup j'ai tout qui est fait avec les api List, ListItem etc.

J'ai l'impression que si on ne le fait pas maintenant, on va jamais le faire même si on dit que. Je trouve ça dommageable de ne pas utiliser les bons composants SURTOUT qu'on les utilisait avant https://github.com/cozy/cozy-settings/pull/453/files#diff-0f7804eafe5f56106581a7149f16bcf316a4466155780612661ed7bffb305fb1L32-L71 . Et donc je vois plus ça comme une régression technique qu'autre chose. Je suis d'avis de corriger maintenant.